src/test/java/organization/OrganizationsTest.java

```diff
@@ -14,13 +20,24 @@
 
     private static String orgId;
 
+    private static Stream<Arguments> createOrganizationData() {
+        return Stream.of(
+                Arguments.of("Test happy path", "Long description of my organization Long description of my organization", "unique_organization", "https://developer.atlassian.com/"),
+                Arguments.of("Need to have at least three letters", "Short description", "abc", "http://localhost:5000"),
+                Arguments.of("Special characters !@#$%^&*()", "desc", "can_have_numbers_too123", "https://google.com"),
+                Arguments.of("Unique name test", "Random desc", "must_be_unique_organization_name", "http://www.cnn.com"));
+    }
+
+    @DisplayName("Create Organization with valid data")
+    @ParameterizedTest(name = "Display name: {0}, desc: {1}, name: {2}, website: {3}")
+    @MethodSource("createOrganizationData")
     @Test
-    public void createNewOrganizationWithAllData() {
+    public void createNewOrganizationWithAllData(String displayName, String desc, String name, String website) {
```

Comment:
może lepsza nawa tego testu to createNewOrganizationWithValidData - wtedy wiadomo, że będziesz przekazywał poprawne dane i będziesz się spodziewał 200.